baseapp/baseapp_test.go

```diff
@@ -39,39 +39,32 @@ var (
 
 type paramStore struct {
 	db *dbm.MemDB
+	t  *testing.T
 }
 
 func (ps *paramStore) Set(_ sdk.Context, key []byte, value interface{}) {
 	bz, err := json.Marshal(value)
-	if err != nil {
-		panic(err)
-	}
-
-	ps.db.Set(key, bz)
+	require.NoError(ps.t, err)
+	err = ps.db.Set(key, bz)
+	require.NoError(ps.t, err)
 }
 
 func (ps *paramStore) Has(_ sdk.Context, key []byte) bool {
 	ok, err := ps.db.Has(key)
-	if err != nil {
-		panic(err)
-	}
-
+	require.NoError(ps.t, err)
 	return ok
 }
 
 func (ps *paramStore) Get(_ sdk.Context, key []byte, ptr interface{}) {
 	bz, err := ps.db.Get(key)
-	if err != nil {
-		panic(err)
-	}
+	require.NoError(ps.t, err)
 
 	if len(bz) == 0 {
 		return
 	}
 
-	if err := json.Unmarshal(bz, ptr); err != nil {
-		panic(err)
-	}
+	err = json.Unmarshal(bz, ptr)
+	require.NoError(ps.t, err)
```

Comment:
Ok, this looks nicer that the previous version. I'd just do a tiny change:

```suggestion
	require.NoError(ps.t, json.Unmarshal(bz, ptr))
```